### DISCUSSION TITLE
How to define TS for access and others thing?

### DISCUSSION THREAD

**Discussion by IRediTOTO** [payloadcms/payload#1788] [Q&A] [answered]
Hi, I am trying define typescript for `access` in `CollectionConfig`
```
const User = CollectionConfig={
...
access: {
    read: ({ req: { user } }) => {  // How to define ts for user var here ?
      return true;
    },
  },
  ...

}
```

**Comment by JessRynkar** [comment, marked as answer]
Hi @IRediTOTO - [this blog post](https://payloadcms.com/blog/announcing-generated-types) should get you on track!

After running `payload generate:types` you should be able to import `user` from your generated types file and use this in your code above.
